packages/block-library/src/image/theme.scss

```diff
@@ -2,6 +2,6 @@
 	@include caption-style-theme();
 }
 
-.wp-block-image {
+:where(.wp-block-image) {
```

Comment:
This caused a regression: there's a default block style that is now being applied to the image block:

https://github.com/WordPress/gutenberg/blob/9606680085131576f87c5731f44156cb67e2a2b2/packages/edit-post/src/classic.scss#L24